app/questionnaire/rules.py

```diff
@@ -44,27 +43,28 @@ def evaluate_goto(goto_rule, metadata, answer_store, group_instance):
     return True
 
 
-def evaluate_repeat(repeat_rule, answer_store):
+def evaluate_repeat(repeat_rule, answer_store, max_repeats=25):
```

Comment:
`max_repeats` should be defined as a constant at the top of this file, rather than passed in, as we never override it.